packages/tpetra/core/example/BlockCrs/Tpetra_TestBlockCrs.cpp

```diff
@@ -238,7 +238,7 @@ int main (int argc, char *argv[])
     // uses a different device. An easy check is device configuration for
     // 4 mpi nodes as the typical number of GPUs per node is 4.
     if (commTest->getRank () < 4) {
-      exec_space::print_configuration (std::cout, false);
+      exec_space{}.print_configuration (std::cout, false);
```

Comment:
What is this change doing?  I don't understand the curly braces.

Reply:
It's constructing an exec_space instance like (), since print_configuration is supposed to be non-static

Reply:
Its the default constructor. I.e. call the member function instead of the static function. The previous thing was actually wrong, just happens to work with most old backends but I think we technically only allow member functions for that. 

Curly braces is just another mode of construction in C++, one which is generally now preferred over ().